hubot-k8s.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  annotations:
+    kompose.cmd: kompose convert
+    kompose.version: 1.11.0 ()
+  creationTimestamp: null
+  labels:
+    io.kompose.service: hubot
+  name: hubot
+spec:
+  replicas: 1
+  strategy: {}
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        io.kompose.service: hubot
+    spec:
+      containers:
+      - args:
+        - bin/hubot
+        - -a
+        - slack
+        env:
+        - name: PORT
+          value: "8000"
+        - name: HUBOT_HEROKU_KEEPALIVE_URL
```

Comment:
Where is this used? Can we remove it? I don’t like the IP being coded there.

Reply:
Looks like `hubot-heroku-keepalive` package is what is looking for this. I say remove it. https://github.com/hubot-scripts/hubot-heroku-keepalive. You can probably find the commit to make sure you remove all artifacts